storybook/.storybook/preview.jsx

```diff
@@ -13,6 +13,7 @@ addParameters({
     name: 'Figma',
```

Comment:
not sure if we fully committed to ditching the Figma panel, but if removing some of the figma links in components is a stride toward that, I'm personally fine with scrapping it all (until/unless we find a good use for it)

Reply:
oops, those are indeed working in the table! my b